eng/common/templates/job/execute-sdl.yml

```diff
@@ -28,7 +28,8 @@ jobs:
     - name: AzDOBuildId
       value: $[ dependencies.setupMaestroVars.outputs['setReleaseVars.AzDOBuildId'] ]
   pool:
-    vmImage: windows-2019
+    name: NetCore1ESPool-Svc-Internal
```

Comment:
normally we should not change anything below `eng/common` folder, are those changes needed?